perf/jarl-perf.csv

```diff
@@ -0,0 +1,4 @@
+label,baseline,perf/a,perf/b
+e3e755d17d,0.5786034939005645,4.833100232853919,7.443960715380243
+c0251ea41f,1.4706713179398083,5.908276495889907,9.236181934405773
+156807007b,0.40935575474398594,5.577915364549673,8.82058527772661
```

Comment:
We'll replace the contents of these CSV files once we can generate that using GitHub actions